sentry_logging/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: sentry_logging
+description: An integration which adds support for recording log from the logging package.
+version: 1.0.0
```

Comment:
I think we should match the versioning of the other packages, it would be 6.2.0 now

Reply:
```suggestion
version: 6.2.0
```